core/src/main/java/org/apache/calcite/sql/validate/SqlValidatorImpl.java

```diff
@@ -5486,23 +5505,35 @@ targetNamespace, castNonNull(catalogReader.unwrap(Prepare.CatalogReader.class)),
       throw newValidationError(createTable, RESOURCE.createTableRequiresAsQuery());
     }
 
-    // Note, this can either a row expression or a query expression with an optional ORDER BY
-    // We're not currently handling the row expression case.
-    // This may or may not be within the scope of what we want
-    // to support by release: (https://bodo.atlassian.net/browse/BE-4478)
+    //Note, this can either a row expression or a query expression with an optional ORDER BY
```

Comment:
This should be `// Note` (keep the space for formatting consistency).

Reply:
Done